assets/js/googlesitekit/modules/datastore/__fixtures__/index.js

```diff
@@ -33,10 +30,9 @@ const alwaysActive = [ 'search-console', 'site-verification' ];
  * @return {Object[]} Array of module objects.
  */
 export function withActive( ...slugs ) {
-	const activeSlugs = alwaysActive.concat( slugs );
 	return modules.map( ( module ) => ( {
 		...module,
-		active: activeSlugs.includes( module.slug ),
+		active: module.forceActive || ( slugs.includes( module.slug ) ),
```

Comment:
Unnecessary extra parens 🙂 
```suggestion
		active: module.forceActive || slugs.includes( module.slug ),
```